### PR TITLE
[CPU] gemm convolution: fixed bias offset

### PR DESCRIPTION
**Pull request by antonvor**
We added the group offset twice to the biases, as a result of which we could get accuracy errors or segfaults. It was noticed on the arm64 platform, because under x64 other primitives are mainly used.

**TODO**
- [x] need to add tests for gemm_convolution primitive with biases

PR to onednn fork: https://github.com/openvinotoolkit/oneDNN/pull/194

### CODE REVIEW

**Comment by maxnick** [comment]
@EgorDuplensky , could you please review?

**Review by EgorDuplensky — COMMENTED** [review]
In general LGTM.
@antonvor As discussed, please double check single_layer_tests with valgrind if we still have unresolved issues after this fix.

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/group_convolution.cpp [hunk 1]
```diff
@@ -383,6 +383,20 @@ INSTANTIATE_TEST_SUITE_P(smoke_GroupConv_1D_Gemm_FP32, GroupConvolutionLayerCPUT
...
+                        ::testing::Combine(
+                                ::testing::Combine(
+                                        groupConvParams_ExplicitPadding_Gemm_1D,
+                                        ::testing::Values(ElementType::f32),
+                                        ::testing::Values(ElementType::undefined),
+                                        ::testing::Values(ElementType::undefined),
+                                        ::testing::ValuesIn(inShapesGemm1D),
+                                        ::testing::Values(CommonTestUtils::DEVICE_CPU)),
+                                ::testing::ValuesIn(filterCPUInfoForDevice(CPUParams_Gemm_1D)),
+                                ::testing::Values(fusingAddPerChannel),
```
Comment (EgorDuplensky) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/group_convolution.cpp:396]: Do we really need to test both gemm_1D and gemm_1D_nspc?
Does it matter from the perspective of the issue we fix?
Applicable to all added instances.

**Review by EgorDuplensky — APPROVED** [review]

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/group_convolution.cpp (hunk 1 above)
Comment (antonvor) [reply src/plugins/intel_cpu/tests/functional/single_layer_tests/group_convolution.cpp:396]: As part of this task, we only need tests for the nspc layout, but I would leave bias tests for the ncsp layout for testing coverage

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**